R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -56,6 +56,16 @@ TableHandle <- R6Class("TableHandle",
       return(c(self$nrow(), self$ncol()))
     },
     
+    merge = function(...) {
+      table_list <- unlist(c(...))
+      if (length(table_list) == 0) {
+        return(TableHandle$new(self$.internal_rcpp_object$merge(NULL)))
```

Comment:
Seems like this should just return self.

Reply:
When I do this, the C++ object being pointed to is the same, so we now have two references to the same object, and modification to either reference will change the underlying thing. This is pretty un-Deephaven.

Reply:
Same issue as the above proposal with merge.

Reply:
`return(self)`.  No reason to touch the c++.